draco/core/containers.py

```diff
@@ -1182,7 +1182,7 @@ class SiderealStream(
             "distributed_axis": "freq",
             "compression": COMPRESSION,
             "compression_opts": COMPRESSION_OPTS,
-            "chunks": (64, 128, 128),
+            "chunks": (32, 512, 2048),
```

Comment:
This is consistent with the daily filtering pipeline, we just never changed the defaults